libs/components/lists/testing/src/modules/paging/paging-harness.ts

```diff
@@ -0,0 +1,121 @@
+import { HarnessPredicate, TestElement } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+
+import { SkyPageControlHarness } from './page-control-harness';
+import { SkyPagingContentHarness } from './paging-content-harness';
+import { SkyPagingHarnessFilters } from './paging-harness-filters';
+
+/**
+ * Harness for interacting with a paging component in tests.
+ */
+export class SkyPagingHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-paging';
+
+  #getNextButton = this.locatorForOptional('li button.sky-paging-btn-next');
+  #getPreviousButton = this.locatorForOptional(
+    'li button.sky-paging-btn-previous',
+  );
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyPagingHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyPagingHarnessFilters,
+  ): HarnessPredicate<SkyPagingHarness> {
+    return SkyPagingHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Clicks the next button.
+   */
+  public async clickNextButton(): Promise<void> {
+    const button = await this.#getNextButton();
+
+    if (button === null) {
+      throw new Error('Could not find the next button.');
+    }
+
+    if (await this.#buttonIsDisabled(button)) {
+      throw new Error(
+        'Could not click the next button because it is disabled.',
+      );
+    }
+
+    await button.click();
+  }
+
+  /**
+   * Clicks the page button with the requested number. Returns an error if that page is not displayed.
```

Comment:
can you change this to say `Throws an error` to match similar documentation in other harnesses.